pyproject.toml

```diff
@@ -48,6 +48,7 @@ pillow ="^10.2.0"
 # model-platforms
 litellm = { version = "^1.38.1", optional = true }
 google-generativeai = { version = "^0.6.0", optional = true }
+ai71 = { version = "^0.0.18", optional = true }
```

Comment:
if we are using OpenAI compatibility then we don't need to add ai71 to our dependency